include/open62541/server.h

```diff
@@ -82,6 +82,7 @@ typedef struct {
 
 struct UA_ServerConfig {
     UA_Logger logger;
+    const UA_Logger *pLogger; /* pointer to logger for easier substitution of logger */
```

Comment:
Rename pLogger to `logging`.
We don't use the p-prefix anywhere else in the SDK.
And we should keep the API stable and not have to rename `pLogger` in the future.

Add a comment to the `logger` member that it is deprecated but still supported at this time.

Reply:
renamed, added comment